lms/services/key.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from jose import constants, jwk
+from sqlalchemy import func
+
+from lms.models import Key
+from lms.services import aes
+
+
+class KeyService:
+    def __init__(self, db):
+        self._db = db
+
+    def generate(self, aes_secret) -> Key:
```

Comment:
This generates a new key. We should have a celery app that rotates them every X hours days.